va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
@xhaihao  Argh, the name of that syntax element changed between standard versions - apologies for the confusion there.

I'm still unclear exactly how this feature can work, though.  Do you have any documentation or a working implementation?  I'm particularly curious about how the reference pictures will be handled.